js_modules/dagit/packages/core/src/sensors/SensorDetails.tsx

```diff
@@ -70,30 +67,8 @@ export const SensorDetails: React.FC<{
     repositoryName: repoAddress.name,
     repositoryLocationName: repoAddress.location,
   };
-  const repo = useRepository(repoAddress);
-  const pipelinesAndJobs = repo?.repository.pipelines;
 
   const latestTick = ticks.length ? ticks[0] : null;
-  const targetCount = targets?.length || 0;
-
-  const targetNames = React.useMemo(
-    () => new Set((targets || []).map((target) => target.pipelineName)),
-    [targets],
-  );
-
-  const anyPipelines = React.useMemo(() => {
-    return (pipelinesAndJobs || []).some(
-      (pipelineOrJob) => !pipelineOrJob.isJob && targetNames.has(pipelineOrJob.name),
-    );
-  }, [pipelinesAndJobs, targetNames]);
-
-  const pipelineOrJobLabel = React.useMemo(() => {
-    if (anyPipelines) {
-      return targetCount > 1 ? 'Jobs / Pipelines' : 'Pipeline';
-    }
-    return targetCount > 1 ? 'Jobs' : 'Job';
-  }, [anyPipelines, targetCount]);
```

Comment:
All of this code was determining whether to show Jobs / Pipelines, Pipelines or Job. In the list view we just say `Asset / Job` and I think doing that in the detail view is better for consistency + much simpler 😅